content/docs/usage/csi-driver-spiffe/README.md

```diff
@@ -159,8 +217,8 @@ ClusterIssuer.
 helm upgrade -i -n cert-manager cert-manager-csi-driver-spiffe jetstack/cert-manager-csi-driver-spiffe --wait \
   --set "app.logLevel=1" \
   --set "app.trustDomain=my.trust.domain" \
-  --set "app.approver.signerName=clusterissuers.cert-manager.io/csi-driver-spiffe-ca" \
```

Comment:
Is it explained anywhere why `app.approver.signerName` isn't meant to be used along with  `app.runtimeIssuanceConfigMap`? Or why this field isn't needed anymore?

Reply:
See "Simpler Install with no signerName" [on the release notes](https://github.com/cert-manager/csi-driver-spiffe/releases/tag/v0.6.0)

I considered documenting this a bit but I think the signerName setting is either an antipattern or else so specifically an extreme power user feature that it wasn't really worth it.

Reply:
Thanks, the link you shared was super useful. You are right, let's just recommend people to not use this flag by not showing it at all in the installation instructions 👍